drv/cosmo-seq-server/src/vcore.rs

```diff
@@ -377,12 +377,16 @@ impl VCore {
         };
 
         let ereport = Ereport {
-            k: "pmbus.alert",
+            k: if !power_good {
+                "hw.pwr_good.bad"
```

Comment:
Do we want to put this in a `hw.power.*` class? Though saying that it leads me to also wonder if pmbus should be a subclass of power or not.